zerver/views/realm_playgrounds.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from django.core.exceptions import ValidationError
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+
+from zerver.decorator import require_realm_admin
+from zerver.lib.actions import do_add_realm_playground, do_remove_realm_playground
+from zerver.lib.request import REQ, JsonableError, has_request_variables
+from zerver.lib.response import json_error, json_success
+from zerver.lib.validator import check_string, check_url
+from zerver.models import RealmPlayground, UserProfile
+
+
+def check_pygments_language(var_name: str, val: object) -> str:
+    s = check_string(var_name, val)
```

Comment:
We should use e.g. `check_capped_string` (I think) -- the ones that check the length.  In general, we want to do as much validation as possible close to the client.

(FWIW we generally avoid Django's validators system, in favor of doing all input validation in `REQ`)